platform/ios/src/MGLMapView.mm

```diff
@@ -6276,6 +6280,12 @@ - (void)updateCalloutView
     }
 }
 
+- (void)updateAttributionAlertView {
+    if (self.attributionController && self.attributionController.presentingViewController) {
```

Comment:
The check for `self.attributionController` is superfluous, since `self.attributionController.presentingViewController` will be NO, when `attributionController` is `nil`.